core/cli/commands.py

```diff
@@ -217,17 +215,21 @@ def save_mod_json(data):
                     }
                 ]
                 if mod_ns == "new":
-                    mod_questions = [{
+                    mod_question = [{
                         'type': 'input',
                         'name': 'mod_ns',
                         'message': 'Name of your namespace',
-                    }] + mod_questions
-                mod_details = prompt(mod_questions)
+                        'validate': NamespaceValidation
+                    }]
+                    mod_namespace = prompt(mod_question)
                 try:
-                    mod_ns = mod_details['mod_ns']
+                    mod_ns = mod_namespace['mod_ns']
                 except:
                     pass
+                #The modules in the selected namespace are static defined to the ModuleNameValidation class.
+                ModuleNameValidation.modules = self.modules[mod_ns].keys() if mod_ns in self.modules.keys() else []
```

Comment:
Can you explain more about this line please?

Reply:
Added a more descriptive comment line.

Reply:
Actually, line below that.